<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QProgressBar#progressBar{
border: 1px solid grey;
border-bottom-right-radius: 7px;
border-bottom-left-radius: 7px; 
border-top-right-radius: 7px;
border-top-left-radius: 7px; 
color: black; 
/*text-align: centre;*/
}

QProgressBar#progressBar::chunk {
background-color: rgb(141, 189, 91);
border-bottom-right-radius: 5px;
border-bottom-left-radius: 5px; 
border-top-right-radius: 5px;
border-top-left-radius: 5px; 
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3" stretch="3,1,5,1,3,1,1">
    <item>
     <layout class="QGridLayout" name="topGridLayout" rowstretch="0,0,0" columnstretch="3,7,1">
      <property name="verticalSpacing">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="selectFilesLabel">
        <property name="text">
         <string>Selected files:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="openDirLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="selectFilesButton">
        <property name="minimumSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Select the pictures you want to rescale or compress.
Supported formats: JPEG, JPG, PNG, BMP, TIFF.</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="destinationFolderLabel">
        <property name="text">
         <string>Destination folder:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="saveDirLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="overWriteCheckBox">
        <property name="toolTip">
         <string>The original images will be overwritten with the edited version.</string>
        </property>
        <property name="text">
         <string>Overwrite the existing files</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="selectDestinationButton">
        <property name="minimumSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Select the folder where the images will be saved.</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer1">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QGroupBox" name="fileNameGroupBox">
      <property name="title">
       <string>Output file names</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="2,2,1,4">
       <item>
        <widget class="QRadioButton" name="originalNameRadioBtn">
         <property name="text">
          <string>Use original file names</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="customNameHorizontalLayout" stretch="7,10">
         <item>
          <widget class="QRadioButton" name="customNameRadioBtn">
           <property name="text">
            <string>Customize file name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nameLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QGridLayout" name="nameAdditifsGridLayout" columnstretch="3,4,2,4,2" columnminimumwidth="0,0,0,0,0">
         <item row="0" column="2">
          <spacer name="fileNameHSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="autoIncrLabel">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="prefixLabel">
           <property name="text">
            <string>Add prefix:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="autoIncrLbl">
           <property name="text">
            <string>Auto increment:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="prefixLineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="suffixLabel">
           <property name="text">
            <string>Add suffix:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="suffixLineEdit"/>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="incrOffsetLabel">
           <property name="text">
            <string>Increment offset:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QSpinBox" name="incrOffsetSpinBox">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>61</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QGroupBox" name="settingsGroupBox">
      <property name="title">
       <string>Scaling and compression parameters</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="settingsGridLayout" rowstretch="1,1,1" columnstretch="2,3,3,1">
         <item row="2" column="1">
          <spacer name="settingsHSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>48</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="3">
          <widget class="QSpinBox" name="qualitySpinBox">
           <property name="toolTip">
            <string>Image is saved to a file with a specified quality factor (in the range 0 to 100).
Specify 0 to obtain highly compressed files, 100 for uncompressed files.
The lower the quality, the smaller the output file size.
Quality &gt; 80 may result in a significant file size increasing.</string>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QSlider" name="qualitySlider">
           <property name="toolTip">
            <string>Image is saved to a file with a specified quality factor (in the range 0 to 100).
Specify 0 to obtain highly compressed files, 100 for uncompressed files.
The lower the quality, the smaller the output file size.
Quality &gt; 80 may result in a significant file size increasing.</string>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QSlider" name="scaleSlider">
           <property name="toolTip">
            <string>Images are resized while maintaining their aspect ratio: their width
and height are multiplied by the scale value divided by 100.</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QSpinBox" name="scaleSpinBox">
           <property name="toolTip">
            <string>Images are resized while maintaining their aspect ratio: their width
and height are multiplied by the scale value divided by 100.</string>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="scaleLabel">
           <property name="text">
            <string>Size scale (%):</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="qualityLabel">
           <property name="text">
            <string>Image quality:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="2" colspan="2">
          <widget class="QCheckBox" name="defaultSettingsCheckBox">
           <property name="text">
            <string>Use the default settings</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="bottomHorizontalLayout">
      <item>
       <widget class="QPushButton" name="exitButton">
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="executeButton">
        <property name="text">
         <string>Resize and Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
